src/NzbDrone.Core/Applications/Lidarr/LidarrV1Proxy.cs

```diff
@@ -102,7 +104,17 @@ public ValidationFailure TestConnection(LidarrIndexer indexer, LidarrSettings se
 
             try
             {
-                Execute<LidarrIndexer>(request);
+                var version = _httpClient.Post<LidarrIndexer>(request).Headers.GetSingleValue("X-Application-Version");
+
+                if (version == null)
+                {
+                    return new ValidationFailure(string.Empty, "Failed to fetch Lidarr version");
+                }
+
+                if (new Version(version) < new Version(MinimumApplicationVersion))
+                {
+                    return new ValidationFailure(string.Empty, $"Lidarr version should be at least 1.0.2. Version reported is {version}", version);
```

Comment:
MinimumApplicationVersion in validation error :)

Reply:
`$"Lidarr version should be at least {MinimumApplicationVersion}. Version reported is {version}")` no?